api/test/wfl/tools/endpoints.clj

```diff
@@ -46,6 +46,13 @@
       (http/get {:headers (auth/get-auth-header)})
       util/response-body-json))
 
+(defn get-workflows-by-status
+  "Query v1 api for all workflows managed by `_workload` with a specific status."
+  [{:keys [uuid] :as _workload} status]
+  (-> (wfl-url "/api/v1/workload/" uuid "/workflows")
```

Comment:
Since you're calling the same endpoint, it might be better to augment the existing method to take an optional status parameter. The signature could look like
```clojure
[{:keys [uuid] :as workload} & [status]] 
```
(this is one way to have one optional parameter in your function declarations)

Reply:
Another FYI is that ^^ is called a variadic function, the other way I've specified optional params is with multi-arity function definitions.  Some more info on both kinds here:

https://theburningmonk.com/2013/09/clojure-multi-arity-and-variadic-functions/

Reply:
Thank you! I went with the variadic route for this simply so that I didn't have duplicated code. Also @ehigham, I added a utils function to trim slashes from the front and back of a url part and added that in a call to map in wfl-url, that way we should be able to leave the endpoint calls as they are and the wfl-url function will clean it up for us. I think this also helps make the url parts more readable with the slashes included.